spec/support/analytics_helper.rb

```diff
@@ -1,6 +1,6 @@
 module AnalyticsHelper
-  def stub_analytics
-    controller.analytics = FakeAnalytics.new
+  def stub_analytics(user: nil)
+    controller.analytics = FakeAnalytics.new(user:)
```

Comment:
Since we didn't end up actually needing this, I'd suggest reverting it for now.

```suggestion
  def stub_analytics
    controller.analytics = FakeAnalytics.new
```

Reply:
AnalyticsHelper module is now back to its previous version.